examples/Makefile

```diff
@@ -328,7 +328,7 @@ ifeq ($(PLATFORM),PLATFORM_DESKTOP)
     ifeq ($(PLATFORM_OS),BSD)
         # Libraries for FreeBSD, OpenBSD, NetBSD, DragonFly desktop compiling
         # NOTE: Required packages: mesa-libs
-        LDLIBS = -lraylib -lGL -lpthread -lm
+        LDLIBS = -lraylib -lGL -lpthread -lm -lglfw
```

Comment:
GLFW shouldn't be required, now is compiled together with raylib in module [rglfw](https://github.com/raysan5/raylib/blob/master/src/rglfw.c).

Reply:
ok